plugins/kontent-plugin/src/components/brand-logo.js

```diff
@@ -0,0 +1,29 @@
+import * as React from "react"
+
+export default function GatsbyContentfulLogo() {
```

Comment:
It doesn't _really_ matter but figured we should update this function name:

```suggestion
export default function GatsbyKontentLogo() {
```